IfcDiscreteAccessory Rail Import BIMQ 2021-07-21T07:41:33.5120880Z (#187)
bSI-InfraRoom/IFC-Specification: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_DiscreteAccessoryTypeBracket/Documentation.md

> Properties of bracket used in railway. The property set can be used by the predefined type BRACKET of IfcDiscreteAccessory

Reviewer: ```suggestion
Properties of bracket used in railway. The property set can be used by the predefined type BRACKET of IfcDiscreteAccessory.
```

> Properties of bracket used in railway. The property set can be used by the predefined type BRACKET of IfcDiscreteAccessory.

Reviewer 2: ```suggestion
Properties of a bracket. The property set can be used by the predefined type BRACKET of IfcDiscreteAccessory.
```
